modules/loop_taxonomy/loop_taxonomy.views_default.inc

```diff
@@ -106,7 +120,7 @@ function loop_taxonomy_views_default_views() {
   $handler->display->display_options['arguments']['term_node_tid_depth']['validate']['type'] = 'taxonomy_term';
   $handler->display->display_options['arguments']['term_node_tid_depth']['depth'] = '0';
   $handler->display->display_options['arguments']['term_node_tid_depth']['break_phrase'] = TRUE;
-  /* Filter criterion: Content: Published or admin */
+  /* Filterkriterie: Indhold: Udgivet eller administrator */
```

Comment:
It looks like the views have been exported with the wrong language and translations has been deleted!